decisions/websre/0002-web-sre-service-documentation.md

```diff
@@ -0,0 +1,48 @@
+# 2. Web SRE Service Documentation
+
+Date: 2021-06-01
+Scope: Web SRE Team
+
+## Status
+
+Proposed
```

Comment:
Will we change this to "Accepted" before merging? I guess we don't really need to record the status in the ADR – if the PR is open, it's "proposed". If the PR gets closed, it's "rejected" and if we merge it, it's "accepted".

Reply:
I've just been adding a commit saynig it was accepted pre PR merge. Open to other ways though